client/batch.go

```diff
@@ -46,7 +45,7 @@ type Batch struct {
 	//   // string(b.Results[0].Rows[0].Key) == "a"
 	//   // string(b.Results[1].Rows[0].Key) == "b"
 	Results    []Result
```

Comment:
i wonder if `Result` is still a useful abstration now that this API is internal.